src/main/java/seedu/duke/parser/ParameterParser.java

```diff
@@ -473,17 +479,28 @@ public static LocalDate parseDateTag(String parameter) throws InputTransactionIn
      */
     public static int parseEntryTag(String parameter) throws MoolahException {
         int index;
+        //@@chinhan99
+        if (containAlphabet(parameter)) { // Checks if parameter contains alphabets
```

Comment:
Wow! That's a stronger check to ensure that the index is strictly integer and no special symbols allowed.